api_app/script_analyzers/observable_analyzers/stratosphere.py

```diff
@@ -0,0 +1,123 @@
+import os
+import logging
+import requests
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import settings
+from datetime import datetime, date
+
+logger = logging.getLogger(__name__)
+
+db_name0 = "stratos_ip_blacklist_last24hrs.csv"
+db_name1 = "stratos_ip_blacklist_new_attacker.csv"
+db_name2 = "stratos_ip_blacklist_repeated_attacker.csv"
+
+db_loc0 = f"{settings.MEDIA_ROOT}/{db_name0}"
+db_loc1 = f"{settings.MEDIA_ROOT}/{db_name1}"
+db_loc2 = f"{settings.MEDIA_ROOT}/{db_name2}"
+
+
+class Stratos(classes.ObservableAnalyzer):
+    def check_in_list(self,dataset_loc, ip):
+        with open(dataset_loc, "r") as f:
+            db = f.read()
+
+        db_list = db.split("\n")
+
+        count = 0
+        for ip_tuple in db_list:
+            if count < 2:
+                count += 1
+                continue
+            else:
+                if ip in ip_tuple:
+                    ip_rating = ((ip_tuple.split(","))[2]).strip()
+                    return(True, ip_rating)
+        return(False, "")
```

Comment:
You could write `return False, ""` and extract it `found, rate = self.check_in_list(db_loc0, ip)`.

Or better, you could avoid to use the boolean value at all. The empty string is already "False".

Reply:
I have removed the boolean value.